Allow multiple welcomes per commit (#754)
mlswg/mls-protocol: draft-ietf-mls-protocol.md

  * Compute an EncryptedGroupSecrets object that encapsulates the `init_secret`
    for the current epoch and the path secret (if present).

* Construct one or more Welcome messages from the encrypted GroupInfo object,
  the encrypted key packages, and any PSKs for which a proposal was included in
  the Commit. The order of the `psks` MUST be the same as the order of
  PreSharedKey proposals in the `proposals` vector.  As discussed on
  {{joining-via-welcome-message}}, the committer is free to choose how many

>   Welcome messages to construct.  However, the set of Welcome messages produced
>   in this step MUST include one Welcome encrypted for each new member added in
>   the Commit.

Reviewer: ```suggestion
  Welcome messages to construct.  However, the set of Welcome messages produced
  in this step MUST cover every new member added in the Commit.
```
I found this sentence quite hard to understand (basically it means that every new member must be able to decrypt one of the welcome messages?), I tried a reformulation
